Makefile

```diff
@@ -38,6 +38,12 @@ test-unit-backend:
 build-backend:
 	go build $(BUILD_OPTS) -o plugin-backend -mod=readonly cmd/plugin-backend.go
 
+# docker inspect --format '{{.Architecture}}' quay.io/redhat-cne/openshift-origin-release:rhel-9-golang-1.23-openshift-4.19
+# base image above has a arm64 archiecture; need to build executable /plugin-backend as amd64 to be match final container archiecture 
+.PHONY: build-backend-dev
```

Comment:
this won't be needed if we use the image from brew. If you need to build locally on a Mac you can use the --platform flag
`podman build --platform=linux/amd64 -t test-dashboards -f Dockerfile.dev` and
`podman run -p 9002:9002 --platform=linux/amd64 test-dashboards:latest`